storage/src/adb/any.rs

```diff
@@ -1,25 +1,32 @@
-//! An authenticatable & mutable key-value store based on an MMR over the log of state-change
-//! operations.
+//! `Any` is an authenticated database (ADB) based on an MMR over a log of state-change operations
```

Comment:
<img width="1015" alt="image" src="https://github.com/user-attachments/assets/3d81f4bb-e1e3-4a6e-8b80-e46afc2d5bb7" />


Including `Any` right away is redundant IMO: `An authenticated database (ADB) that provides succinct proofs of _any_ value ever associated with a key.`

I'd explain that it use an MMR and Journal in the following lines (linking to both with `crate::Journal/MMR`?

Reply:
done